index.ts

```diff
@@ -1,33 +1,26 @@
-import { AuditsClient } from './src/audits';
 import { FronteggAuthenticator } from './src/authenticator';
-import { IdentityClient } from './src/identity';
-import { contextResolver, withAuthentication, contextResolverWithPermissions } from './src/identity';
-import { ContextHolder, frontegg, fronteggNextJs, IFronteggOptions } from './src/middleware';
-import { NotificationsClient } from './src/notifications';
-import { FronteggPermissions } from './src/permissions';
-import { RbacMiddleware } from './src/rbac';
-import { SsoClient } from './src/sso';
-import { TenantsClient } from './src/tenants';
-import { AuthzClient } from './src/authz';
-import { HttpClient } from './src/http';
+import { ContextHolder } from './src/components/context';
+import { withAuthentication } from './src/middlewares';
+import {
+  AuditsClient,
+  NotificationsClient,
+  TenantsClient,
+  SsoClient,
+  IdentityClient,
+  AuthzClient,
+  HttpClient,
+} from './src/clients';
 
-export * from './src/events';
+export * from './src/clients/events';
```

Comment:
check what is the usage here and if customers are using it. I know it was used for the connectivity feature which is currently deprecated and maybe for the webhook feature